docs/compatibility.md

```diff
@@ -64,6 +64,20 @@ is generally equivalent to
 client = httpx.Client(**kwargs)
 ```
 
+The HTTPX equivalent of `prepare_request` of `requests.Session` instance is `build_request` of `httpx.Client`.
+
+## Request arguments
+
+The `Request` class of `httpx` takes the arguments
```

Comment:
IMO we should avoid duplicating this information and instead link to the Developer Interface docs for `Request`.

Reply:
agree.